app/controllers/v2/content_items_controller.rb

```diff
@@ -4,7 +4,10 @@ def index
       content_format = params.fetch(:content_format)
       fields = params.fetch(:fields)
       publishing_app = params[:publishing_app]  # can be blank
-      render json: Queries::GetContentCollection.new(content_format: content_format, fields: fields, publishing_app: publishing_app).call
+      render json: Queries::GetContentCollection.new(content_format: content_format,
```

Comment:
style niggle: move `content_format:` onto a new line indented by only four spaces, line up the following lines to match.